pkg/minikube/kubeconfig/config_test.go

```diff
@@ -128,6 +173,120 @@ func TestSetupKubeConfig(t *testing.T) {
 	}
 }
 
+func TestGetKubeConfigStatus(t *testing.T) {
+
+	var tests = []struct {
+		description string
+		ip          net.IP
+		existing    []byte
+		err         bool
+		status      string
+	}{
+		{
+			description: "empty ip",
+			ip:          nil,
+			existing:    fakeKubeCfg,
+			err:         true,
+		},
+		{
+			description: "no minikube cluster",
+			ip:          net.ParseIP("192.168.10.100"),
+			existing:    fakeKubeCfg,
+			status:      "Kubeconfig does not have a record of a minikube cluster",
```

Comment:
Once you change the statuses to not be hardcoded strings then these checks will be much cleaner and robust.  You'll lose some of the specificity of why the error occurred, but I think thats ok since it should be clear from the test.